test/synapse/2019-06-01/azartifacts/pollers.go

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package azartifacts
+
+// TODO replace this code with data-plane specific definitions
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"net/http"
+	"net/url"
+	"time"
+)
+
+// CreateDataFlowDebugSessionResponsePoller provides polling facilities until the operation completes
+type CreateDataFlowDebugSessionResponsePoller interface {
+	Done() bool
+	Poll(ctx context.Context) (*http.Response, error)
+	FinalResponse(ctx context.Context) (*CreateDataFlowDebugSessionResponseResponse, error)
+	ResumeToken() (string, error)
+}
+
+type createDataFlowDebugSessionResponsePoller struct {
+	// the client for making the request
+	pipeline azcore.Pipeline
+	pt       pollingTracker
```

Comment:
Where is pollingTracker defined?  Should it be deleted?

Reply:
Deleted. Done.